radio/src/targets/horus/flyskyHallStick_driver.cpp

```diff
@@ -31,9 +31,20 @@ unsigned char HallCmd[264] __DMA;
 
 STRUCT_HALL HallProtocol = { 0 };
 STRUCT_HALL HallProtocolTx = { 0 };
+signed short hall_raw_values1[FLYSKY_HALL_CHANNEL_COUNT];
+signed short hall_raw_values2[FLYSKY_HALL_CHANNEL_COUNT];
```

Comment:
For increased code readability, these variables can possibly get more intuitive names, like hall_raw_values_last or previous or another name in this vain.